src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/GeneratedComInterfaceAttributeData.cs

```diff
@@ -35,6 +35,7 @@ public static GeneratedComInterfaceData From(GeneratedComInterfaceCompilationDat
     internal sealed record GeneratedComInterfaceCompilationData : InteropAttributeCompilationData
     {
         public ComInterfaceOptions Options { get; init; } = ComInterfaceOptions.ManagedObjectWrapper | ComInterfaceOptions.ComObjectWrapper;
+        public INamedTypeSymbol? ExceptionToUnmanagedMarshaller { get; init; }
```

Comment:
I don't think this should ever be nullable. If the user didn't supply it we should be able to contruct the default one instead and avoid using a `INamedTypeSymbol`.

Reply:
I disagree. Coming from the attribute data itself, there's no good named type symbol to put here for the default case. What we use depends on the return type, and `null` is the value of the field in the attribute at runtime (if someone were to call GetAttribute).

Reply:
I see. This looks like an implementation detail I'm missing. If `null` makes sense I'd hope we don't see `null` after `GeneratedComInterfaceCompilationData` is processed.

Reply:
This is converted from a Roslyn symbol to our data model in `CalculateStubInformation` so from that point onwards there's no `null`.